src/pages/gas.tsx

```diff
@@ -0,0 +1,533 @@
+// Third-party libraries
+import React, { ComponentPropsWithRef } from "react"
+import { graphql, PageProps } from "gatsby"
+import { GatsbyImage } from "gatsby-plugin-image"
+import { useTranslation } from "gatsby-plugin-react-i18next"
+import {
+  Box,
+  BoxProps,
+  Flex,
+  FlexProps,
+  Heading,
+  HeadingProps,
+  Link,
+  ListItem,
+  Table,
+  Tbody,
+  Text,
+  Th,
+  Thead,
+  Td,
+  Tr,
+  UnorderedList,
+} from "@chakra-ui/react"
+
+// Internal libraries
+import { getImage } from "../utils/image"
+
+// Components
+import ButtonLink from "../components/ButtonLink"
+import Callout from "../components/Callout"
+import Card from "../components/Card"
+import Emoji from "../components/Emoji"
+import ExpandableCard from "../components/ExpandableCard"
+import FeedbackCard from "../components/FeedbackCard"
+import GhostCard from "../components/GhostCard"
+import HorizontalCard from "../components/HorizontalCard"
+import InfoBanner from "../components/InfoBanner"
+import InlineLink from "../components/Link"
+import PageHero from "../components/PageHero"
+import PageMetadata from "../components/PageMetadata"
+import Pill from "../components/Pill"
+import Translation from "../components/Translation"
+
+const Content = (props: BoxProps) => <Box px={8} w="full" {...props} />
+
+const Divider = (props: BoxProps) => (
+  <Box my={16} w="10%" h={1} bg="homeDivider" {...props} />
+)
+
+const Page = (props: FlexProps) => (
+  <Flex
+    width="full"
+    direction="column"
+    align="center"
+    my={0}
+    mx="auto"
+    {...props}
+  />
+)
+
+export const StyledCard = (props: ComponentPropsWithRef<typeof Card>) => (
+  <Card
+    flex="1 1 30%"
+    minW="280px"
+    maxW={{ base: "full", lg: "46%" }}
+    p={6}
+    {...props}
+  />
+)
+
+const H2 = (props: HeadingProps) => (
+  <Heading fontSize={{ base: "2xl", md: "2rem" }} lineHeight={1.4} {...props} />
+)
+
+const H3 = (props: HeadingProps) => (
+  <Heading
+    as="h3"
+    fontSize={{ base: "xl", m: "2xl" }}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const GasPage = ({ data }: PageProps<Queries.GasPageQuery>) => {
+  const { t } = useTranslation()
+
+  const benefits = [
+    {
+      emoji: "🪪",
+      description:
+        "Gas keeps Ethereum sybil-resistant by preventing malicious actors from overwhelming the network with fraudulent activities.",
+    },
+    {
+      emoji: ":money_with_wings:",
+      description:
+        "Because computation costs gas, spamming Ethereum with expensive transactions, either accidentally and maliciously, is financially disencentivised.",
+    },
+    {
+      emoji: ":hourglass_flowing_sand:",
+      description:
+        "A hard-limit on the amount of computation that can be done at any one time prevents Ethereum from being overwhelmed, helping to ensure the network is always accessible.",
+    },
+  ]
+
+  const heroContent = {
+    title: "Network fees",
+    header: "Network fees",
```

Comment:
Just fyi, we will have an `h1` and `h2` with the same content.

Perhaps we could call the title just `gas`? the same as in the url path. That will help with the search since someone who is searching for "gas" will most likely find this page first.

Reply:
Nice catch, meant to fix. Updated.